src/kup/__main__.py

```diff
@@ -170,16 +171,35 @@ def get_package_metadata(package: GithubPackage) -> PackageMetadata:
 
 
 # build a rich.Tree of inputs for the given package metadata
-def package_metadata_tree(p: Union[PackageMetadata, Follows], lbl: Union[str, None] = None) -> Tree:
+def package_metadata_tree(
+    p: Union[PackageMetadata, Follows], lbl: Union[str, None] = None, show_health: bool = False
+) -> Tree:
     if lbl is None:
         tree = Tree('Inputs:')
     else:
         rev = f' - github:{p.org}/{p.repo}' if type(p) == PackageMetadata else ''
         follows = (' - follows [green]' + '/'.join(p.follows)) if type(p) == Follows else ''
-        tree = Tree(f'{lbl}{rev}{follows}')
+        health = ''
+        if show_health and type(p) == PackageMetadata:
+            auth = {'Authorization': f'Bearer {os.getenv("GH_TOKEN")}'} if os.getenv('GH_TOKEN') else {}
+            commits = requests.get(f'https://api.github.com/repos/{p.org}/{p.repo}/commits', headers=auth)
+            if commits.ok:
+                commits_list = [c['sha'] for c in commits.json()]
+                if p.rev in commits_list:
+                    idx = commits_list.index(p.rev)
+                    if idx == 0:
+                        health = ' 🟢 up to date'
+                    elif idx < 5:
+                        health = f' 🟠 {idx} versions behind master'
+                    else:
+                        health = f' 🔴 {idx} versions behind master'
+            else:
+                print(commits.json())
+
+        tree = Tree(f'{lbl}{rev}{follows}{health}')
```

Comment:
If there's an easy way to align the circles, consider trying that out. (I'm not sure it would look better, but it might be worth a try.)